chia/pools/pool_wallet.py

```diff
@@ -495,11 +494,6 @@ async def generate_fee_transaction(self, fee: uint64, coin_announcements=None) -
         )
         return fee_tx
 
-    async def publish_transactions(self, travel_tx: TransactionRecord, fee_tx: Optional[TransactionRecord]):
-        await self.wallet_state_manager.add_pending_transaction(travel_tx)
-        if fee_tx is not None:
-            await self.wallet_state_manager.add_pending_transaction(dataclasses.replace(fee_tx, spend_bundle=None))
-
```

Comment:
I think you should leave this pattern in.  You fixed the problem where the spend bundles were not being aggregated before being pushed, but as you identified in your comment, this means that the standard wallet has no transaction record for the fee leaving.  If you keep this method, that problem should be solved.  This is a pattern we already use for CATs, Offers, and other things still in development.

Reply:
We broadcast all unconfirmed TXs to the full node though, so that is incompatible with this pattern. Is there a way to create an unconfirmed TX but not broadcast it?

Reply:
Wouldn't it be simpler to create the TX when the first on gets confirmed instead?

Reply:
We don't broadcast anything to a full node if the SpendBundle slot is None, that's why `dataclasses.replace(fee_tx, spend_bundle=None)` is in there

Reply:
The way that the transaction book-keeping really works was very very not-obvious to me - I'd really appreciate a comment in the above function.